native/sapp-windows/src/gl.rs

```diff
@@ -598,5 +602,11 @@ gl_loader!(
     fn glGenVertexArrays(n: GLsizei, arrays: *mut GLuint) -> (),
     fn glFrontFace(mode: GLenum) -> (),
     fn glCullFace(mode: GLenum) -> (),
-    fn glGenTextures(n: GLsizei, textures: *mut GLuint) -> ()
+    fn glGenTextures(n: GLsizei, textures: *mut GLuint) -> (),
```

Comment:
The new gl generator is so much better compared to the nonsense we had before, isn't it? :D

Reply:
Yes, it was a pleasure to do this change with the new loader.